stonesoup/assignment/assignAlgs.py

```diff
@@ -0,0 +1,313 @@
+import numpy
+
+
+def assign2D(C, maximize=False):
```

Comment:
I think this should be move to `stonesoup.dataassociator._assignment` as not intend to be public component.

Also would be good to get in are usual documentation format, but not essential.